src/common/offline_transformations/src/compress_quantize_weigths.cpp

```diff
@@ -220,8 +222,93 @@ ov::pass::CompressQuantizeWeights::CompressQuantizeWeights() {
         return true;
     };
 
-    auto m = std::make_shared<ov::pass::pattern::Matcher>(fq_pattern, "CompressQuantizeWeights");
-    this->register_matcher(m, callback);
+    auto m = std::make_shared<pattern::Matcher>(fq_pattern, "CompressWeightsWithFakeQuantize");
+    register_matcher(m, callback);
+}
+
+static std::shared_ptr<ov::op::v0::Constant> get_fake_convert_shift(
+    const std::shared_ptr<ov::op::v13::FakeConvert>& fake_convert) {
+    if (fake_convert->get_input_size() < 3)
+        return nullptr;
+    const auto shift = ov::as_type_ptr<ov::op::v0::Constant>(fake_convert->get_input_node_shared_ptr(2));
+    if (!shift)
+        return nullptr;
+    float value = -1.0f;
+    if (!ov::op::util::get_single_value(shift, value) || value != 0.0f)
```

Comment:
`value != 0.0f` I think is always false, can be removed?

Reply:
Why do you think it's always true?

Reply:
`get_single_value` call will update "value"